src/components/sbb-journey-summary/sbb-journey-summary.tsx

```diff
@@ -26,6 +26,10 @@ export class SbbJourneySummary implements ComponentInterface {
    * The RoundTrip prop. This prop controls if one or two arrows are displayed in the header.
    */
   @Prop() public roundTrip?: boolean;
+
+  /** Heading level of the journey header element (e.g. h1-h6). */
+  @Prop() public headerLevel?: InterfaceSbbJourneySummaryAttributes['level'] = '3';
```

Comment:
Naming should be consistent
```suggestion
  @Prop() public headerLevel?: InterfaceSbbJourneySummaryAttributes['headerLevel'] = '3';
```